src/extensions/lightbox-controls/test/lightbox-controls.cypress.js

```diff
@@ -45,7 +45,7 @@ describe( 'Test CoBlocks Lightbox Controls extension', function() {
 			helpers.addBlockToPost( 'core/image', true );
 
 			helpers.upload.imageToBlock( 'core/image' );
-
+			cy.wait( 500 );
```

Comment:
What was this? I think the need for this was fixed by commit https://app.circleci.com/pipelines/github/godaddy-wordpress/coblocks/6222/workflows/84aa207a-2dbd-4e71-8f55-57b614e6e7b7

Reply:
Feel free to remove it, then. It was failing for me in local when not waiting enough time for the upload.